labgrid/driver/power/phoenix_fl_switch.py

```diff
@@ -0,0 +1,80 @@
+'''
+This Driver was tested on a FL SWITCH 2303-8SP1 with FW-version 3.27.01 BETA
+file   phoenix_fl_switch.py
+author Raffael Krakau
+date   2023-08-24
+
+Copyright 2023 JUMO GmbH & Co. KG
+'''
+
+from telnetlib import Telnet
```

Comment:
The build node currently has Python 3.10.12 - 
```
 Deprecated since version 3.11, will be removed in version 3.13: The [telnetlib](https://docs.python.org/3/library/telnetlib.html#module-telnetlib) module is deprecated (see [PEP 594](https://peps.python.org/pep-0594/#telnetlib) for details and alternatives).
 ```

Vermutlich wäre https://pypi.org/project/telnetlib3/ eine alternative :thinking:

Reply:
switched to pexpect.spawn(…"telnet…")